frontend/kesaseteli/youth/src/pages/_document.tsx

```diff
@@ -0,0 +1,3 @@
+import ServerDocument from 'shared/server/ServerDocument';
+
+export default ServerDocument;
```

Comment:
And also this if you prefer
```suggestion
export { default from ServerDocument } from 'shared/server/ServerDocument';
```